public/assets/css/style.css

```diff
@@ -1,3 +1,23 @@
-body {
-    color: #f76c6c;
-}
\ No newline at end of file
+:root {
+    --primary-color: #181515;
```

Comment:
pas sûr que ce soit très utile... autant le faire direct dans le 
[data-theme="light"],:root:not([data-theme="dark"])